src/index.ts

```diff
@@ -109,40 +103,30 @@ export function extractProperties<T>(object?: T) {
   }
 }
 
-export function createSyncFn<T extends AnyAsyncFn>(
-  workerPath: string,
-  bufferSize?: number,
-  timeout?: number,
-): Syncify<T>
-export function createSyncFn<T extends AnyAsyncFn>(
-  workerPath: string,
-  options?: SynckitOptions,
-): Syncify<T>
-export function createSyncFn<R, T extends AnyAsyncFn<R>>(
+export function createSyncFn<T extends AnyAsyncFn<R>, R = unknown>(
   workerPath: string,
```

Comment:
I removed these signatures because after removing `bufferSize` the linter wanted me to unify the signatures. so I improved the only signature to map properly to existing usages